### DISCUSSION TITLE
Custom Colors won't work... yes I have read other discussions

### DISCUSSION THREAD

**Discussion by nanosar90** [tailwindlabs/tailwindcss#3753] [Help]
Please help. I am trying to use custom colors in Tailwind and have already installed... configured and gotten Tailwind to work in my project. I then added the custom colors into Tailwind and ran 'npm run build:css' from my package.json file and it runs successfully but I cannot get the colors to work. I have tried putting quotes around the colors and not the values and it still doesn't work. Here is my code.
module.exports = {
  theme: {
    extend: {
      colors: {
        limegreen: {
          '50':  '#FBFCF7',
          '100': '#F8FBE1',
          '200': '#EEF69E',
          '300': '#DCEC53',
          '400': '#A8D619',
          '500': '#65DC21',
          '600': '#429E04',
          '700': '#357C06',
          '800': '#295B09',
          '900': '#20450A',
        },
      }
    }
  }
}

**Comment by NovusOpus** [comment]
You need to add this: `const colors = require('tailwindcss/colors')`

**Full config:**
```
const colors = require('tailwindcss/colors')

module.exports = {
  theme: {
    extend: {
      colors: {
        limegreen: {
          50: '#FBFCF7',
          100: '#F8FBE1',
          200: '#EEF69E',
          300: '#DCEC53',
          400: '#A8D619',
          500: '#65DC21',
          600: '#429E04',
          700: '#357C06',
          800: '#295B09',
          900: '#20450A',
        },
      },
    },
  },
  variants: {},
  plugins: [],
}
```

You can check a working example here: [Tailwind Play link](https://play.tailwindcss.com/K49T3N1WZd)

**Reply by nanosar90** [reply]
I didn't mention it, but I do have that at the top of my tailwind.config.js file. I thought just adds their extended color pallette. Not custom ones?

**Reply by NovusOpus** [reply]
> I didn't mention it, but I do have that at the top of my tailwind.config.js file. I thought just adds their extended color pallette. Not custom ones?

Oh I see, Well it's working fine with either case, as you can see in the Tailwind Labs Play, By keeping `const colors = require('tailwindcss/colors')` for curated colors or without it.

**Reply by NovusOpus** [reply]
> I didn't mention it, but I do have that at the top of my tailwind.config.js file. I thought just adds their extended color pallette. Not custom ones?

Can you share your HTML code, and what Env you using to build your CSS? Also did you check if the generated css file contains your colors?

**Reply by nanosar90** [reply]
I am using VS code and building the CSS through my tailwind.config.js and the npm. They aren't showing up in my linked styles file. The file is linked properly. So why they aren't showing up idk, which explains why when I type "bg-limegreen-500" nothing works. I am newer to coding and have only been doing it for 2-3 months (UW FSF Javascript coding bootcamp) so I apologize if I am bad at answering your questions. I appreciate the help.

**Reply by simonswiss** [reply]
Hola 👋

The best way to get help is to share the code you're working with. If this is private stuff you cannot show, try and recreate an example of the same setup (often called a "minimal reproduction" or "repro") so people can have a look and troubleshoot for you.

It's super hard to make a guess about what could be the issue based on what you shared so far, because your Tailwind config file is actually correct and should work properly.

Looking at the whole setup, we could see things like what your entry CSS file looks like, what the `build:css` script actually does, etc.

**Comment by simonswiss** [comment]
Hey @nanosar90 - what you have here should definitely work. Pasting your exact code in a Tailwind Play immediately gives me access to the new `lime-green` values:

![image](https://user-images.githubusercontent.com/485747/111249462-dfd18f00-865f-11eb-904f-e661fbdc1dc6.png)

If you are able to share a GitHub repo with your code where this is not working, I'll have a look for you 👍
